lnwire/peer_storage.go

```diff
@@ -0,0 +1,73 @@
+package lnwire
+
+import (
+	"bytes"
+	"encoding/binary"
+	"fmt"
+	"io"
+)
+
+// MaxPeerStorageBytes is maximum size in bytes of blob in peer storage message.
+const MaxPeerStorageBytes = 65531
+
+var ErrPeerStorageBytesExceeded = fmt.Errorf("peer storage bytes exceede" +
+	"d")
+
+// PeerStorageBlob is the type of the data sent by peers to other peers for
+// backup.
+type PeerStorageBlob []byte
```

Comment:
This might be one of the few times that I actually think you *don't* want to newtype this. Since we are dealing with truly opaque bytes, I don't see the value of wrapping it with a newtype.

Reply:
The way of encoding this is different from normal bytes in the `readElement` function. We write the length of the byte first then the byte if I understand the spec correctly.